src/core/Relayer.sol

```diff
@@ -0,0 +1,50 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.25;
+
+import {IERC20} from "@forge-std/interfaces/IERC20.sol";
+import {SafeTransferLib} from "../vendor/SafeTransferLib.sol";
+
+contract Relayer {
+    using SafeTransferLib for IERC20;
+
+    event RelayerAction(); // Graffiti for bridging operations through Relayer
```

Comment:
I think we probably need some more information in this event. Can you ask @Ktl-XV or @AndresElizondo ?

Reply:
Also @byeongminP - do you think it would be useful to add something e.g. Relay's `requestId` to this event or we're okay with getting it from the calldata?

For @sohai - do you think there's some piece of information that would be useful here for watching the events for milestone 2 with destination swaps? 

I think overall we only really care about the event itself as all the data is available from elsewhere, but maybe there's something that I'm missing

Reply:
I think adding auxiliary indexed params to the event is not worth the extra gas cost in this case. Considering that we'd eventually want to cache quote params (intents, etc) for destination swaps, we can get away with storing the `requestId` extracted from the origin tx calldata off-chain.